src/app/containers/Figure/index.jsx

```diff
@@ -20,30 +20,29 @@ const FigureContainer = ({
   src,
   height,
   width,
-}) => (
-  <Figure>
-    <ImagePlaceholder ratio={ratio}>
-      <RequestContextConsumer>
-        {({ platform }) =>
-          platform === 'amp' ? (
-            <AmpImg
-              alt={alt}
-              attribution={copyright || ''}
-              layout="responsive"
-              src={src}
-              height={height}
-              width={width}
-            />
-          ) : (
-            <Image alt={alt} src={src} width={width} />
-          )
-        }
-      </RequestContextConsumer>
-      {renderCopyright(copyright)}
-    </ImagePlaceholder>
-    {renderCaption(captionBlock)}
-  </Figure>
-);
+}) => {
+  const { platform } = useContext(RequestContext);
+  return (
+    <Figure>
+      <ImagePlaceholder ratio={ratio}>
+        {platform === 'amp' ? (
```

Comment:
Now that the context has been moved to the top of this function, we should check to see whether the whole `<Figure>` including caption, copyright & placeholder re-render when a platform context value is updated, or whether it is clever enough to only re-render the actual Image part of this container.

Reply:
You're correct about this usecase in that it's an unlikely user journey. 

Where we pull in translations via the Service Context, e.g. https://github.com/bbc/simorgh/pull/1656/files#diff-5192a5db9928faa2474d9f458c25d32cR36, we don't want to re-render the whole component every time a particular translation changes. We may need to use the useMemo hook to ensure a re-render only happens when relevant values are updated.